src/HPND.xml

```diff
@@ -37,9 +37,9 @@
       </p>
       <optional>
         <p>
-            <alt match=".*" name="copyrightHolder2">&lt;copyright holder&gt;</alt> DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING
+            <alt match=".*" name="copyrightHolder2">&lt;copyright holder&gt;</alt> <alt match+"DISCLAIMS|DISCLAIM" name="disclaim">DISCLAIMS</alt> ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING
```

Comment:
There is typo: `match+` instead of `match=`. 
And I would suggest changing it to: `match="DISCLAIMS?"` - this means that the character (S in this case) before the question mark is optional

Reply:
Thanks! fixed

Reply:
fixed this, but still failing...